text/user.css

```
@@ -343,13 +351,22 @@ img[src*="equaliser"] {
 .search-searchCategory-contentArea {
     --carousel-start-chevron-gradient: var(--spice-main);
     --carousel-end-chevron-gradient: var(--spice-main);
 }
+.main-genericButton-button,
+.x-sortBox-sortDropdown,
+.x-filterBox-expandButton,
+.x-filterBox-filterInput,
+.main-nowPlayingView-mainWrapper,
+.main-entityHeader-metaDataText {
+    --text-subdued: var(--spice-subtext) !important;
+    color: var(--spice-subtext) !important;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)